CMakeLists.txt

```diff
@@ -73,15 +73,12 @@ endif()
 #------------------------------------------------------------------------------#
 option(Legion_USE_GASNet "Enable the distributed GASNet backend" OFF)
 if(Legion_USE_GASNet)
-  set(GASNet_THREADING "par" CACHE STRING "GASNet Threading model to use")
-  set_property(CACHE GASNet_THREADING PROPERTY STRINGS seq par parsync)
-  mark_as_advanced(GASNet_THREADING)
+  set(GASNet_THREADING par)
+  set(GASNet_PREFERRED_CONDUITS gemeni aries ibv)
```

Comment:
Good catch on the typo.  Aries and Gemini should be mutually exclusive.  It's still just the default configuration subject to user change.  If you do have both available then chances are likely you'll want to explicitly set the desired conduit anyways.  I don't mind re-ordering since I need to fix the typo anyways but I don't believe it will be an issue that comes up.